cg/apps/demultiplex/notes.md

```diff
@@ -0,0 +1,107 @@
+We loop over all flowcell directories and check if sequencing and transfering is done. This is indicated by the two
```

Comment:
This would be great in Atlas!

Reply:
Will move it when demux is ready for production (next PR)